pkg/client/cmd/clientcmd.go

```diff
@@ -0,0 +1,229 @@
+package cmd
```

Comment:
is this file a straight move?  If not, highlight what isn't for me.

Reply:
Yes, the entire file (`pkg/cmd/util/clientcmd/clientcmd.go`) was moved here, except for its `EnvVars` helper, which was moved as an unexported helper in `pkg/cmd/server/origin/controller/config.go`